web-registry/src/components/organisms/CreditClassForms/CreateCreditClassForm.tsx

```diff
@@ -0,0 +1,93 @@
+import { Formik } from 'formik';
+
+import SaveFooter from 'web-components/lib/components/fixed-footer/SaveFooter';
+import NotFound from 'web-components/lib/components/not-found';
+import RotationalGrazing from 'assets/rotational-grazing.png';
+
+import { useMultiStep } from 'components/templates/MultiStepTemplate';
+import { CreditClassForm } from './CreditClassForm';
+import { CreditClassReview } from './CreditClassReview';
+import { CreditClassFinished } from './CreditClassFinished';
+import { mockTxHash, mockClassId } from './CreditClass.mocks';
+
+import type { CreditClassValues } from './CreditClassForm';
+
+export const createCreditClassSteps = [
+  {
+    id: 'create-credit-class',
+    name: 'Create Credit Class',
+    title: 'Create Credit Class',
+  },
+  {
+    // TODO: ID probably isn't needed on the next two, as they aren't stored in
+    // localstorage?
+    id: '',
+    name: 'Review',
+    title: 'Review',
+  },
+  {
+    id: '',
+    name: 'Finished',
+    title: 'Finished',
+  },
+];
+
+/** must be used within a `MultiStep` context  */
+export const CreateCreditClassForm = (props: {
+  onSubmit: (values: CreditClassValues) => void;
+}): JSX.Element => {
+  const {
+    data,
+    percentComplete,
+    activeStep,
+    handleBack,
+    handleNext,
+    isLastStep,
+  } = useMultiStep<CreditClassValues>();
+
+  const disabledFields: string[] = [];
+  if (!!data.admin.length) disabledFields.push('admin');
+  if (!!data.fee.length) disabledFields.push('fee');
```

Comment:
This was actually just a temporary solution, which I over-engineered because I only wanted to disable the fields if they were receiving data, but I realize now that's not obvious - especially without some TODO comment. For now, I changed to hard-coded value with a todo